mixins/AuthenticationMixin.ts

```diff
@@ -61,8 +62,12 @@ export default class AuthenticationMixin extends Vue {
     }
     const loading = this.$vs.loading()
     try {
-      await authStore.signUpWithEmail({ email, password });
-      this.delayedSendToHome(loading);
+      this.$router.push("/auth/login");
+      await authStore.signUpWithEmail({ email, password, name });
+      loading.close();
+      this.$vs.notification({title:"accountCreated", color: "success"})
+      location.reload();
```

Comment:
Redirecting before the sign up is pretty clever but why are you doing location.reload()? it makes it so that the notification doesn't show because the page is reloaded. Also the notification doesn't have a message, i think the property should be `message` not `title`. Also here is where you should handle the error notification to tell the user whats up, see my comment below.